{
  "comments": [
    {
      "key": {
        "uuid": "9ad45d7e_b31dd668",
        "filename": "cryptfs.c",
        "patchSetId": 2
      },
      "lineNbr": 3597,
      "author": {
        "id": 2936
      },
      "writtenOn": "2015-03-09T08:02:10Z",
      "side": 1,
      "message": "You\u0027re going to exit_err right after, this is still being treated as an error...",
      "revId": "3499c36781ff7ff5243be0c171f763352dffc69e",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    }
  ]
}